server/utilities/distance-utils.js

```diff
@@ -0,0 +1,15 @@
+export function getDistanceFromLatLonInKm(lat1,lon1,lat2,lon2) {
```

Comment:
Comment code, and credit http://stackoverflow.com/questions/27928/calculate-distance-between-two-latitude-longitude-points-haversine-formula